types/providers.d.ts

```diff
@@ -72,6 +72,7 @@ export type OAuthProviderType =
   | "FACEIT"
   | "FortyTwo"
   | "Foursquare"
+  | "Freshbooks"
```

Comment:
Can we also test the types for this new provider [here](https://github.com/nextauthjs/next-auth/blob/a72a5d78a9a66f65c8fe59d4054f2ad45eb491da/types/tests/providers.test.ts#L259)? 😊